modules/templates/proc-adding-templates.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-adding-templates_{context}"]
+= Adding an existing template to {product} by using the Template Editor
+
+You can add an existing template to your {product} instance by using the Template Editor.
+
+.Prerequisites
+
+* You have created a directory that contains template files.
+* You have created at least one template YAML file in the directory.
+
+.Procedure
+
+. Access the Template Editor by using one of the following options:
++
+image::rhdh/template-editor.png[Template Editor]
+** Open the URL `\https://<rhdh_url>/create/edit` for your {product} instance.
+** Click *Create...* in the navigation menu of the {product} console, then click the overflow menu button and select *Template editor*.
+. Click *Load Template Directory*. This allows you to upload a template from a local directory, by navigating to the directory and clicking *Select*.
+. A *Let site view files?* pop-up appears. Click *View files*.
+. In the *Name ** field, enter a unique name for your template.
```

Comment:
What you describe starting from this item is specific to the template loaded. As you cannot assume what the user will load, then remove such steps: `name`, `Owner`, `Repository Location`

Reply:
If I use gitlab instead of github the requested fields are the same:
![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/40172997/525db4e7-1ac0-4e2a-80c9-867324522c4f)

I think we can assume that an owner and repository location will be required for any repository type. Let's have QE verify this, since I cannot test them all. I think it's better to include this information for completeness and add additional details for other types beyond github and gitlab later if we have more details, rather than leaving this information out entirely.